components/Question/index.jsx

```diff
@@ -0,0 +1,59 @@
+import React, { useState, useEffect } from 'react';
+// import React from 'r'
+import Button from './styles';
+
+
+const Question = ({
+    handleAnswer,
+    data: { question, correct_answer, incorrect_answers } }) => {
+    // Combines correct/incorrect answer arrays
+    const combinedArray = [correct_answer, ...incorrect_answers];
+
+    // Helper function to randomize position of correct/incorrect answers in question array
+    function shuffledArray(arr){
```

Comment:
This is a cool helper function but I think it should live in src/utilities as it could be used for any array that is passed into the function, and is not specific to the Question component.